manifest.json

```diff
@@ -63,13 +64,12 @@
       },
       {
         "name": "firmware_nonfree",
-        "choice": ["yes", "no"],
+        "type": "boolean",
```

Comment:
Are we sure that this is propagated on internet cube scripts that feed argument to call this app install ? :D

Reply:
good catch. will check.

Reply:
I put the `firmware_nonfree` back as string as this is not priority and I don't have time to validate everything.